nvflare/lighter/utils.py

```diff
@@ -146,3 +146,29 @@ def sh_replace(src, mapping_dict):
     for k, v in mapping_dict.items():
         result = result.replace("{~~" + k + "~~}", str(v))
     return result
+
+
+def update_storage_locations(local_dir: str,
+                             workspace: str,
+                             default_resource_name: str = "resources.json.default",
+                             job_storage_name: str = "jobs-storage",
+                             snapshot_storage_name: str = "snapshot-storage"):
+    default_resource = f"{local_dir}/{default_resource_name}"
+    target_resource = f"{local_dir}/resources.json"
+    job_storage = f"{workspace}/{job_storage_name}"
+    snapshot_storage = f"{workspace}/{snapshot_storage_name}"
```

Comment:
logic LGTM, we have these strings defined in

```
./nvflare/apis/fl_constant.py
```

But lighter should be before apis, so we can think about how to deal with these common strings later.